src/foam/nanos/auth/LanguageChoiceView.js

```diff
@@ -62,8 +62,8 @@ foam.CLASS({
   }
   ^ .popUpDropDown > div {
     margin-left: -16px;
+    margin-right: -16px;
```

Comment:
Is there a particular reason we are using negative margins here?

Reply:
Due to the paddings of the divs I used the negative margins to properly align the inner divs.